keypoints/include/pcl/keypoints/uniform_sampling.h

```diff
@@ -40,7 +40,7 @@
 #pragma once
 
 #ifdef __DEPRECATED
```

Comment:
I think we can remove `__DEPRECATED` for this file.

Also, something to be done about it, in general? https://github.com/PointCloudLibrary/pcl/pull/3931#issuecomment-615465110

Reply:
@SergioRAgostinho @taketwo

Reply:
Ping ping @SergioRAgostinho @taketwo

Reply:
I don't see any reason why we can't remove it in this file.

Regarding the other instance of it in `visualization/vtk.h`: looks like a rain dance that might be solving some unobvious issue.

Reply:
https://github.com/PointCloudLibrary/pcl/commit/aa4f7e3960bc6ee1c662cc4c0321ddd429f49496

>  disabled those pesky deprecations from VTK 

I'd keep this, just in case, unless you want to dive into the world of pesky VTK deprecations :wink:

Reply:
Nope, I'm good :laughing: